internal/pkg/cli/pipeline_delete.go

```diff
@@ -63,42 +72,50 @@ func newDeletePipelineOpts(vars deletePipelineVars) (*deletePipelineOpts, error)
 	if err != nil {
 		return nil, fmt.Errorf("default session: %w", err)
 	}
+	ssmStore := config.NewSSMStore(identity.New(defaultSess), ssm.New(defaultSess), aws.StringValue(defaultSess.Config.Region))
+	prompter := prompt.New()
 
 	opts := &deletePipelineOpts{
 		deletePipelineVars: vars,
 		prog:               termprogress.NewSpinner(log.DiagnosticWriter),
-		prompt:             prompt.New(),
+		prompt:             prompter,
+		sel:                selector.NewConfigSelect(prompter, ssmStore),
 		secretsmanager:     secretsmanager.New(defaultSess),
 		pipelineDeployer:   cloudformation.New(defaultSess),
 		ws:                 ws,
+		store:              ssmStore,
 	}
 
 	return opts, nil
 }
 
 // Validate returns an error if the flag values passed by the user are invalid.
```

Comment:
nit: can we update the comment to say that `Validate` is for any optional fields?